sso-plugin/optimize-sso-keycloak/README.md

```diff
@@ -0,0 +1,80 @@
+# Camunda Optimize Keycloak SSO Example
+
+This example demonstrates how you can enable SSO with Optimize and Keycloak.
+
+The most important part from the Optimize perspective of this example is the custom Optimize SSO Plugin in:
+[./optimize/sso-keycloak-example-plugin/](./optimize/sso-keycloak-example-plugin/)
+To find out more about the Optimize SSO Plugin mechanism, find the docs [here](https://docs.camunda.org/optimize/latest/technical-guide/import/plugins/#single-sign-on)
+
+It includes a docker-compose with:
+1. Camunda Optimize
+2. ElasticSearch
+3. Camunda BPM
+2. Keycloak Authentication Server
+3. Keycloak Proxy Server
+
+The Keycloak Server has one user:
+
+```
+demo:notdemo
+```
+
+# How to run?
+
+## Clone the Repo
+
+## Download and Configure Optimize
+
+1. Download https://camunda.org/enterprise-release/optimize/2.3.0/camunda-optimize-2.3.0-standalone.zip to folder ./optimize/
```

Comment:
no need for linking the alpha2 release here? Just thinking that this would be useful if we link this example from the alpha  release blog post but then we may forget to update it later on..